### ISSUE TITLE
version 1.0.1 is giving error when using non-user credentials

### ISSUE THREAD

**Issue by mmmorris1975** [mmmorris1975/aws-runas#17]
```
[CRIT] ▶ Error getting IAM user info: ValidationError: Must specify userName when calling with non-User credentials
    status code: 400, request id: 0eeb7d5c-812f-11e8-bc4a-e96a850ff866
```

Likely coming from the iam.GetUser call change in 1.0.1 when aws-runas is called using instance-profile credentials

**Comment by mmmorris1975** [comment]
See 13b90e7a

**Comment by mmmorris1975** [comment]
fixed in 1.0.2
